[1.20.x] Updating italian localisation

BlackShadow77 opened the pull request:
Starting to work on new updates.

_Hello @JMilamber, happy to see you back!_

BlackShadow77 committed:
Update it_it.json

Starting localisation on new updates

BlackShadow77 committed:
Update it_it.json

Starting work on new updates

BlackShadow77 commented:
@JMilamber i need some context on row 86-87

BlackShadow77 committed:
Update it_it.json

BlackShadow77 committed:
translations and adaptations

Keep translating and modified some old translation to fit the new style

JMilamber commented:
Spawner Markers are used when generating the tower to correctly place spawners and Data Marker are used for non-tower containers (chest/barrels etc)
So a block that marks the position of spawners and containers respectively.

Shouldn't be seen outside of the creative menu or the mod breaking.

BlackShadow77 committed:
Traslating it file

Modified some word to better fit the concept; keep translating new lines and clean up blank rows

BlackShadow77 committed:
Changed adaptation

Changed adaptation of sky minion

BlackShadow77 committed:
Changed adaptation

Changed adaptation of sky minion

BlackShadow77 commented:
Thank you for the kind answer, like usual. 
And what about lines 186-192? Are they titles of songs?

JMilamber commented:
Your welcome, thanks for updating the translations!
Yep, 186 -192 is "Author: Song name"  so those don't need translation. :)

BlackShadow77 committed:
Merge branch 'BlackShadow77-patch-2' of https://github.com/BlackShadow77/Brass_Amber_BattleTowers into BlackShadow77-patch-2

BlackShadow77 committed:
Update it_it.json

Changed translation for a word

BlackShadow77 commented:
Hi again.

I did not forgot, translations ready, just taking some times for the final touch. 
Regarding _"core"_ tower, can you give me more contex? I'm trying to crate a more consistency in the italian translations.

Also, i'll add a new line of comments to separate music disks from other to translate, I'll push the commit soon so you can better understand and, eventually, disagree

BlackShadow77 committed:
Changed land localisation

Updated Land localisation to better fit the thematic

BlackShadow77 committed:
Added new comment in en-us file

Added new row for comments to better understand distinction between music disc and others tooltips

BlackShadow77 committed:
localisation improvements

Update some translation to fit the context

BlackShadow77 committed:
updated it_ch with latest translations

JMilamber commented:
No worries, Just let me know once your happy with it. 

Music disc comment: adding separation there is semantics so if it helps its fine with me. 

Core tower:
    Set underground down in deepslate with lots of lava/magma
   The tower is one of the first towers ever built and the core golem there is the only man-made golem (I believe, might be a better question for Brass)
  The "Core" here refers most simply to the core of the earth and its temperature, rather than that it is the "center" of the towers

BlackShadow77 committed:
Completed translations

Completed translations and changing ":" separator with "-" in section 8b

BlackShadow77 commented:
Hi, @JMilamber, i think i'm ready untill next update or idea i'll mark it as pull request so you can proceed.
Have a nice day and don't disappear again!

JMilamber commented:
Sounds good, Merging now :)

You too! im doing my best not too!
